v0/destinations/mautic/data/MauticIdentifyConfig.json

```diff
@@ -0,0 +1,104 @@
+[
+   
+    {
+      "sourceKeys": "email",
+      "destKey": "email",
+      "sourceFromGenericMap": true
+    },
+    {
+      "sourceKeys": "title",
+      "destKey": "title"
+    },
+    {
+      "sourceKeys": "context.traits.title",
+      "destKey": "title"
+    },
+    {
+      "sourceKeys": "originalTimestamp",
+      "destKey": "last_active",
+      "sourceFromGenericMap": true
+    },
+    {
+        "sourceKeys":"firstName",
+        "destKey":"firstname",
+        "sourceFromGenericMap": true
+
+    },
+    {
+        "sourceKeys":"lastName",
+        "destKey":"lastname",
+        "sourceFromGenericMap": true
+    },
+    {
+        "sourceKeys":"phone",
+        "destKey":"phone",
+        "sourceFromGenericMap": true
+    },
+    {
+        "sourceKeys":"city",
+        "destKey":"city",
+        "sourceFromGenericMap": true
+    },
+    
+    {
+        "sourceKeys":"employeeCount",
+        "destKey":"company_size",
+        "sourceFromGenericMap": true
+    },
+    {
+        "sourceKeys":"role",
+        "destKey":"role",
+        "sourceFromGenericMap": true
+    },
+    {
+        "sourceKeys":"website",
+        "destKey":"website",
+        "sourceFromGenericMap": true
+    },
+    {
+        "sourceKeys":"IP",
+        "destKey":"IP",
+        "sourceFromGenericMap": true
+    },
+    {
+        "sourceKeys":"timezone",
+        "destKey":"timezone",
+        "sourceFromGenericMap": true
+    },
+    {
+        "sourceKeys":"subscriptionStatus",
+        "destKey":"subscription_status",
+        "sourceFromGenericMap": true
```

Comment:
Please edit this. This should not be added to generic Field mapping. Reason being it is a property specific to this destination only.